daliuge-engine/test/apps/test_plasma.py

```diff
@@ -63,7 +65,7 @@ def test_plasma(self):
 
         a = FileDROP('a', 'a', filepath=in_file)
         b = MSPlasmaWriter('b', 'b')
-        c = PlasmaDROP('c', 'c')
+        c = PlasmaDROP('c', np.random.bytes(20))
```

Comment:
...and then this wouldn't be necessary anymore

Reply:
Do you think this would ok:

`   object_id = dlg_string_param('object_id', None)
    plasma_link = dlg_string_param('plasma_link', '/tmp/plasma')

    def initialize(self, **kwargs):
        object_id = self.uid
        if len(self.uid) != 20:
            object_id = np.random.bytes(20)
        if self.object_id is None:
           self.object_id = object_id

    def getIO(self):
        return PlasmaIO(plasma.ObjectID(self.object_id), self.plasma_link)

    @property
    def dataURL(self):
        return "plasma://%s" % (plasma.ObjectID(self.object_id))`

Reply:
@rtobar I think this is right. https://github.com/ICRAR/daliuge/pull/30/commits/e0f74878aa557774a9e8966b2ef1fb94782d25ca